README.md

```diff
@@ -16,14 +16,37 @@ Mongo-hydra is an attempt to open source all that learning in a proper reuseable
 
 ## Release Schedule
 
-|Version |  Component / Feature                | Github Milestone                   |
-|--------|-------------------------------------| -------------------------------------------------------------- |
-|`v0.0.1`| Basic scaffolding + connection logic| [v0.0.1](https://github.com/unumotors/mongo-hydra/milestone/1) |
-|`v0.0.2`| Manage replication                  | [v0.0.2](https://github.com/unumotors/mongo-hydra/milestone/2) |
-|`v0.0.3`| Sharding                            | [v0.0.3](https://github.com/unumotors/mongo-hydra/milestone/3) |
-|`v0.0.4`| User and roles                      | [v0.0.4](https://github.com/unumotors/mongo-hydra/milestone/4) |
-|`v0.0.5`| Authentication                      | [v0.0.5](https://github.com/unumotors/mongo-hydra/milestone/5) |
-|`v0.0.6`| K8s Operator + CRDs                 | [v0.0.6](https://github.com/unumotors/mongo-hydra/milestone/6) |
+| Version  | Component / Feature                  | Github Milestone                                               |
+|----------|--------------------------------------|----------------------------------------------------------------|
+| `v0.0.1` | Basic scaffolding + connection logic | [v0.0.1](https://github.com/unumotors/mongo-hydra/milestone/1) |
+| `v0.0.2` | Manage replication                   | [v0.0.2](https://github.com/unumotors/mongo-hydra/milestone/2) |
+| `v0.0.3` | Sharding                             | [v0.0.3](https://github.com/unumotors/mongo-hydra/milestone/3) |
+| `v0.0.4` | User and roles                       | [v0.0.4](https://github.com/unumotors/mongo-hydra/milestone/4) |
+| `v0.0.5` | Authentication                       | [v0.0.5](https://github.com/unumotors/mongo-hydra/milestone/5) |
+| `v0.0.6` | K8s Operator + CRDs                  | [v0.0.6](https://github.com/unumotors/mongo-hydra/milestone/6) |
+
+## Comparison
+
+The prometheus operators [KubeDB](https://kubedb.com/docs/0.9.0/concepts/databases/mongodb/) and [Percona Kubernetes Operator](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/index.html) exist. This lists their current feature set.
+
+|                 |                    | KubeDB              | Percona                   |
```

Comment:
can we adda  column for mongo-hydra and maybe switch to emojis or icons like ⛔   ✅